cmd/ocsp-updater/main.go

```diff
@@ -318,16 +410,17 @@ func main() {
 		dbMap, err := sa.NewDbMap(c.OCSPUpdater.DBConnect)
 		cmd.FailOnError(err, "Could not connect to database")
 
-		cac, closeChan := setupClients(c, stats)
+		cac, pubc, closeChan := setupClients(c, stats)
 
 		updater, err := newUpdater(
-			auditlogger,
 			stats,
 			clock.Default(),
 			dbMap,
 			cac,
+			pubc,
 			// Necessary evil for now
 			c.OCSPUpdater,
+			len(c.Publisher.CT.Logs),
```

Comment:
If we're going to reference this config section from both the Publisher and the OCSP Updater, we should hoist it to its own config section, e.g. CT or Common.CT.